src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 spring.application.name=gateway

```

Comment:
What's with the "^M" on this file? I'm not sure what it means and if it should be there.

Reply:
I think they are enconding issues

Reply:
Fixed. Was the line delimiters configuration problem.